src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
```

Comment:
- [ ] Kindly add at least 1 child component for the `Calculator.js` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component).